src/ProgramBlockEditor.js

```diff
@@ -362,9 +363,17 @@ class ProgramBlockEditor extends React.Component<ProgramBlockEditorProps, Progra
                         <FormattedMessage id='ProgramBlockEditor.programHeading' />
                     </h2>
                     <div className='ProgramBlockEditor__options'>
-                        <AddNodeToggleSwitch
-                            isAddNodeExpandedMode={this.state.addNodeExpandedMode}
+                        <ToggleSwitch
+                            ariaLabel={this.props.intl.formatMessage({id:'ProgramBlockEditor.toggleAddNodeExpandMode'})}
+                            value={this.state.addNodeExpandedMode}
                             onChange={this.handleChangeAddNodeExpandedMode}
+                            contentsTrue={<AddIcon />}
+                            contentsFalse={<AddIcon />}
+                            className={
+                                this.state.addNodeExpandedMode ?
+                                    'ProgramBlockEditor__addNode ProgramBlockEditor__addNode--checked' :
```

Comment:
I think `ProgramBlockEditor__add-node-toggle-switch` would be better for the class name as this is for the switch and not for one of the add nodes themselves.